telegram/messageid.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# pylint: disable=R0903
```

Comment:
nitpicking just b/c I went over those recently: please check, if we actually need this here :)